### ISSUE TITLE
Unsupported Audio Sample Rate

### ISSUE THREAD

**Issue by aaronson2012** [paul-gauthier/aider#612]
### Issue

aider was crashing when i ran the /voice command and I realized that my computers microphone did not support a sample rate of 16kHz. It was an easy fix for me i just changed the sample_rate variable to one my mic supported but this isn't a very elegant solution. Could there be a better way so all mics are supported? 

### Version and model info

Latest verision

**Comment by paul-gauthier** [comment]
Thanks for trying aider and filing this issue. Aider solved this issue in 73c6af8851.

The change is available in the main branch. You can get it by installing the latest version from github:

```
python -m pip install --upgrade git+https://github.com/paul-gauthier/aider.git
```

If you have a chance to try it, let me know if it works better for you.

**Comment by paul-gauthier** [comment]
I'm going to close this issue for now, but feel free to add a comment here and I will re-open or file a new issue any time.
